Pipelines/sync-pipeline-repo.yml

```diff
@@ -67,7 +67,7 @@ stages:
         displayName: 'Push ${{parameters.BranchToCreate}} to AzDO repo'
 
         # Set default org and project
-      - powershell: az devops configure --defaults organization=$(System.CollectionUri) project="$(System.TeamProject)"
+      - powershell: az devops configure --defaults organization=$(System.CollectionUri) project=$(System.TeamProject)
```

Comment:
Hello @mikefactorial , could you check if this change is intentional? It would undo my change from PR #46 . Maybe there was some overlap when this PR was created and my change was merged into vnext?

Reply:
Sorry, I just realized this PR was closed without merging it into vnext, so all is good :)